docs/cas-server-documentation/installation/Configuration-Properties.md

```diff
@@ -3760,8 +3760,14 @@ RESTful settings for this feature are available [here](Configuration-Properties-
 
 If AUP is controlled via JDBC, decide how choices should be remembered back inside the database instance. Database settings for this feature are available [here](Configuration-Properties-Common.html#database-settings) under the configuration key `cas.acceptableUsagePolicy.jdbc`.
 
+See source code of `AcceptableUsagePolicyProperties.java` for more detailed explainations of the settings.
```

Comment:
Remove this line please.

Reply:
Okay. I would have found this reference generally helpful, because from a user-only perceptive I would not know to look in the sources (locating the correct file can be a struggle already).